packages/storybook/src/web-components/alert/alert.stories.tsx

```diff
@@ -0,0 +1,240 @@
+
+export const Playground: Story = {
+  ...AlertTemplate,
+  name: 'Playground',
+  args: {
+    children: textTemplate(),
+    type: 'ok',
+  },
+  parameters: {
+    docs: {
+      sourceState: 'shown',
+    },
+  },
+  tags: ['!autodocs'],
+};
+
+export const Info: Story = {
+  ...AlertTemplate,
+  name: 'Info',
+  args: {
+    type: 'info',
+    children: textTemplate('Info alert'),
+  },
+};
+
+export const Ok: Story = {
+  ...AlertTemplate,
+  name: 'OK / Success',
+  args: {
+    type: 'ok',
+    children: textTemplate('Okay alert'),
+  },
+};
+
+export const Warning: Story = {
+  ...AlertTemplate,
+  name: 'Warning',
+  args: {
+    type: 'warning',
+    children: textTemplate('Warning alert'),
+  },
+};
+
+export const Error: Story = {
+  ...AlertTemplate,
+  name: 'Error',
+  args: {
+    type: 'error',
+    children: textTemplate('Error alert'),
+  },
+};
+
+// TODO: Remove comment tags when Heading is done
+// And add a Story to the MDX
+/*export const Heading: Story = {
+  ...AlertTemplate,
+  name: 'With a Heading',
+  args: {
+    type: 'info',
+    children: textTemplate('Alert with heading'),
+  },
+  render: (args) => (
+    <LuxAlert {...args}>
+      <LuxHeading level={2}>
+        Title of the Alert
+      </LuxHeading>
+      <LuxParagraph>{args.children}</LuxParagraph>
+    </LuxAlert>
+  ),
+};*/
+
+export const Icon: Story = {
+  name: 'Custom Icon',
+  args: {
+    type: 'info',
+    children: textTemplate('Alert with custom icon'),
+  },
+  parameters: {
+    docs: {
+      description: {
+        story: 'Een custom icon kan geplaats worden via `slot="icon"`.',
+      },
+    },
+  },
+  render: (args) => (
+    <LuxAlert {...args}>
+      <LuxParagraph>{args.children}</LuxParagraph>
+      <IconMail slot="icon" />
+    </LuxAlert>
+  ),
+};
+
+export const NoIcon: Story = {
+  ...AlertTemplate,
+  name: 'No Icon',
+  args: {
+    type: 'info',
+    hideIcon: true,
+    children: textTemplate('No Icon alert'),
+  },
+};
+
+export const AriaRole: Story = {
+  ...AlertTemplate,
+  name: 'Aria Role Alert',
+  args: {
+    type: 'error',
+    children: 'Houston, we hebben een probleem.',
+    role: 'alert',
+  },
+  parameters: {
+    docs: {
+      description: {
+        story: 'Hint: test met een Screen reader',
+      },
+    },
+  },
+  render: (args: any) => {
+    const [shown, toggleAlert] = useState(false);
+
+    const onClick = () => {
+      toggleAlert(!shown);
+    };
+
+    return (
+      <>
+        {shown ? (
+          <LuxAlert {...args}>
+            <LuxParagraph>{args.children}</LuxParagraph>
+          </LuxAlert>
+        ) : null}
+        <br />
+        <LuxButton appearance="subtle-button" onClick={onClick}>
+          Toggle Alert
+        </LuxButton>
+      </>
+    );
+  },
+};
+
+export const DesignTokens = createDesignTokensStory(meta);
+
+const VisualAlertTemplate = ({ children, type }: PropsWithChildren<JSX.LuxAlert>) => (
```

Comment:
Om consistent te blijven met wat we al hebben: dit graag verplaatsen naar `./visual`

Reply:
Done